DateTimeFormatter.php

```diff
@@ -75,6 +76,26 @@ public function getDiffMessage($count, $invert, $unit)
         return $this->doGetDiffMessage($count, $invert, $unit);
     }
 
+    /**
+     * Returns a DateTime instance for the given datetime
+     *
+     * @param  mixed $datetime
+     *
+     * @return DateTimeInterface
+     */
+    public function getDatetimeObject($datetime = null)
```

Comment:
It's a new method, so we can "fix" things:

```suggestion
    public function getDateTimeObject($datetime = null): DateTimeInterface
```